Source/webserver/src/main/resources/static/js/main.js

```diff
@@ -16,6 +21,29 @@ window.leafletMap = L.map('map', {zoomControl: false}).setView([57.6900727722877
         position: 'bottomright'
     }));
 
+function toggleLocationDropdownMenu()
+{
+    document.getElementById("location-dropdown-menu").classList.toggle("show");
+}
+
+window.onclick = function(event)
+{
+    if(!event.target.matches('.location-dropdown-button'))
+    {
+        var dropdowns = document.getElementsByClassName("location-dropdown-content");
+        
+        var i;
+        for(i = 0; i < dropdowns.length; i++)
+        {
+            var openDropdown = dropdowns[i];
+            if(openDropdown.classList.contains("show"))
+            {
+                openDropdown.classList.remove("show");
+            }
+        }
+    }
+}
```

Comment:
Using jQuery would make it match the rest of the code. Also this would've been unnecessary if a select element was used.

Reply:
Jag är öppen för förslag, hur ungefär tänkte du? Jag är inte världsbäst på webbutveckling som du ser, och ja, koden är lite här och var, men det fungerar i varje fall.

Reply:
@mishaelse, nvm, lyckades lösa det!